src/app-layout/drawer/index.tsx

```diff
@@ -117,16 +168,56 @@ export function Drawer({
         className={clsx(styles['drawer-content'], contentClassName)}
       >
         {!isMobile && regularOpenButton}
-        <TagName aria-label={mainLabel} aria-hidden={!isOpen}>
+        {activeDrawer?.resizable && !isMobile && (
+          <div className={splitPanelStyles['slider-wrapper-side']}>{resizeHandle}</div>
+        )}
+        <TagName aria-label={activeDrawer?.ariaLabels.content || mainLabel} aria-hidden={!isOpen}>
           <CloseButton
             ref={toggleRefs.close}
             className={closeClassName}
-            ariaLabel={closeLabel}
-            onClick={() => onToggle(false)}
+            ariaLabel={activeDrawer?.ariaLabels.closeButton || closeLabel}
+            onClick={() => {
+              onToggle(false);
+              drawers?.onChange({ activeDrawerId: undefined });
+            }}
           />
           {children}
         </TagName>
       </div>
     </div>
   );
 }
+export function DrawerTriggersBar({ isMobile, topOffset, bottomOffset, drawers, contentClassName }: DrawerTriggersBar) {
+  return (
+    <div
+      className={clsx(styles.drawer, styles['drawer-closed'], testutilStyles['drawer-closed'], {
+        [styles['drawer-mobile']]: isMobile,
+      })}
+    >
+      <div
+        style={{ top: topOffset, bottom: bottomOffset }}
+        className={clsx(styles['drawer-content'], styles['non-interactive'], contentClassName)}
+      >
+        {!isMobile && (
+          <aside aria-label="Drawers" className={styles['drawer-triggers']}>
```

Comment:
Hard-coded text. Need to expose a property for that (or have a reminder to not forget before final release)

Reply:
Sure, I can copy what we do for `Tools`. I was under the impression we only did that because it could either be Tools or Navigation, and here it will always be Drawers.